docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx

```diff
@@ -1,76 +1,419 @@
 ---
-title: SSH login approval via Pager Duty
-description: How to configure SSH login approval using PagerDuty and Teleport
-h1: SSH Login Approval using PagerDuty
-videoBanner: RzTB4Lsm09M
+title: Access Requests with PagerDuty
+description: How to set up Teleport's PagerDuty plugin for privilege elevation approvals.
 ---
 
-This guide will talk through how to setup Teleport with PagerDuty. Teleport's
-PagerDuty integration allows you to treat Teleport access and permission
-requests as PagerDuty incidents, notify the appropriate team, and approve
-or deny the requests via PagerDuty special action.
+With Teleport's PagerDuty integration, engineers can access the infrastructure
+they need to resolve incidents quickly—without longstanding admin permissions
```

Comment:
```suggestion
they need to resolve incidents quickly — without longstanding admin permissions
```

Reply:
We don't have a standard around using em dashes vs. en dashes with spaces, but the use of an em dash is more common in US English.